### DISCUSSION TITLE
"Maintance OnGoing" Message is displayed untill internet is not connected to device

### DISCUSSION THREAD

**Discussion by arshad-trusshealth** [mdn/mdn-community#731] [Platform] [answered]
**Description**
When the internet connection is lost, the MDN Web Docs website header displays a "Maintenance Ongoing" message. This message persists as long as the device is disconnected from the internet. Once the connection is restored, the message disappears.

**Steps to Reproduce:**
Open the MDN Web Docs website.
Disconnect your device from the internet.
Observe the header where a "Maintenance Ongoing" message is displayed.
Reconnect to the internet, and the message disappears.
Expected Behavior:
The header should show a message indicating that the site is offline or there should be no maintenance message when there is no connection.

**Actual Behavior:**
"Maintenance Ongoing" message is displayed when there is no internet connection, which might cause confusion.

**Screenshot:**
with internet connected normal navbar

![image](https://github.com/user-attachments/assets/6302302b-5de7-458d-b275-ab0a4cdd198a)

without internet connected message is displayed as 

![image](https://github.com/user-attachments/assets/b1360e8b-e425-40ff-9650-0ea9a72e8a1b)

It may seems like website is undergoing maintance even though website is available and user is not connect to internet.
in my opinion we need to show better message when user is not connected to internet.

**Environment:**
Browser: Chrome Browser Version 128.0.6613.120 
Operating System: Windows 10

**Comment by pransh15** [comment]
Hi @arshad-trusshealth 👋 
Thanks for reporting this. It actually fits better as an issue so our devs can work on the bug, instead of a Discussion.

Can you please [file an issue here](https://github.com/mdn/yari/issues/new?assignees=&labels=bug&projects=&template=bug-report.yml)?

**Reply by arshad-trusshealth** [reply, marked as answer]
hi @pransh15 
Sure I am reporting this as an issue, I am ready work on this let me know how can i do that.
@pransh15  i have reported this above issue with my another account as this account is no longer use please find link to issue below
 
https://github.com/mdn/yari/issues/11842

**Comment by bsmth** [comment]
Thanks for reporting. If there's a yari issue to track it, we can close this Discussion. Thank you :)
